sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/EventPosition.cs

```diff
@@ -219,6 +188,27 @@ public override int GetHashCode()
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override string ToString() => base.ToString();
 
+        /// <summary>
+        ///   Corresponds to the event in the partition at the provided offset.
+        /// </summary>
+        ///
+        /// <param name="offset">The offset of an event with respect to its relative position in the partition.</param>
+        /// <param name="isInclusive">If true, the event at the <paramref name="offset"/> is included; otherwise the next event in sequence will be received.</param>
+        ///
+        /// <returns>The position of the specified event.</returns>
+        ///
+        private static EventPosition FromOffset(string offset,
+                                                bool isInclusive = false)
```

Comment:
Maybe we should default it to `true` here as well? That won't make a difference in this scenario, though.

Reply:
You know... that's a good point.  We shouldn't assume a default here, I think.